jena-benchmarks/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.apache.jena</groupId>
+    <artifactId>jena</artifactId>
+    <version>4.9.0-SNAPSHOT</version>
+    <relativePath>..</relativePath>
+  </parent>
+
+  <name>Apache Jena - Benchmark Suite</name>
+  <artifactId>jena-benchmarks</artifactId>
+  <version>4.9.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <description>Suite with different benchmarks.
+  </description>
+
+  <url>https://jena.apache.org/</url>
+
+  <organization>
+    <name>Apache Jena</name>
+    <url>https://jena.apache.org/</url>
+  </organization>
+
+  <licenses>
+    <license>
+      <name>Apache 2.0 License</name>
+      <url>https://www.apache.org/licenses/LICENSE-2.0</url>
+    </license>
+  </licenses>
+
+  <modules>
+    <module>jena-benchmarks-jmh</module>
```

Comment:
I'm having an IDE build problem.

Code in `TestGraph...` classes depends on a package rooted at `shadedJena480`

e.g.
```
private List<org.apache.shadedJena480.graph.Triple> triples480;
```
which Eclipse isn't finding.

Reply:
I have the same problem with InteliJ IDEA. 
I'm not sure, but it seems to be a general problem with shaded libraries, as I had the same problem with jena-shaded-guava. 
My solution was to manually add "jena\jena-benchmarks\jena-benchmarks-shadedJena480\target\jena-benchmarks-shadedJena480-4.9.0-SNAPSHOT.jar" as a library to jena-bechmarks-jmh.
I had to do the same before with jena-shaded-guave.
--> However, I would be grateful for a real solution to this problem.

Reply:
I find that simply not including the projects with shading works. We can see how this goes for everyone when merged.